src/Form/Form.js

```diff
@@ -587,11 +588,11 @@ class Form {
         const config = getInputConfig(input)
 
         const shouldDecorate = await config.shouldDecorate(input, this)
-        if (!shouldDecorate) return this
+        if (!shouldDecorate && !this.shouldShowCredentialsImportFlow) return this
```

Comment:
I think this should be moved upstream. We do most of the feature flag checks in [`Settings.canAutofillType`](https://github.com/duckduckgo/duckduckgo-autofill/blob/b629ed523510bd0e02611733f45a4e33f6ab4410/src/Settings.js#L257). This way, at this point `shouldDecorate` is already false and you don't have to do anything else here.

Reply:
I moved the check in `Settings.canAutofillType` https://github.com/duckduckgo/duckduckgo-autofill/pull/631/commits/03366ad416641e15bf246cddafef8d9b5cbee473